pyanaconda/ui/gui/spokes/advstorage/dasd.py

```diff
@@ -42,7 +42,8 @@ class DASDDialog(GUIObject):
     def __init__(self, data, storage):
         GUIObject.__init__(self, data)
         self.storage = storage
-        self.dasd = self.storage.dasd
+        self.dasd = [d for d in self.storage.devices if d.type == "dasd"]
+        self.dasd.sort(key=lambda d: d.name)
```

Comment:
Shouldn't there be some better way to do this. I mean removing the `storage.dasd` without proper replacement looks somehow strange to me.
Could you please confirm that this is right way how to get DASD disks now, @vpodzime @dwlehman ?